templates/scenarios/csharp/command-and-response/teamsapp.yml.tpl

```diff
@@ -1,3 +1,4 @@
+# yaml-language-server: $schema=https://developer.microsoft.com/json-schemas/teams-toolkit/teamsapp-yaml/1.0.0/yaml.schema.json
```

Comment:
Does toolkit uses *resource/yaml.schema.json* to check format?
If so, what if *resource/yaml.schema.json* has difference from this remote one?
User may see error hint but actually it passes the TTK check, or user see no error but it fails the TTK check.

Reply:
Currently, the bundled schema is used for schema validation.  This one is only for intellisense.
We have to carefully keep them in sync, or it's a logic error.
If there's a schema change, a new schema  and a new TTK will be published.

Reply:
So for each stable release we can keep them consistency. But will it be a problem if an older TTK opens a newer project, or a newer TTK opens an older project?